openff/interchange/smirnoff/_nonbonded.py

```diff
@@ -328,6 +331,7 @@ def _get_charges(
 
                     total_charge: Quantity = numpy.sum(parameter_value)
                     # assumes virtual sites can only have charges determined in one step
+
                     charges[topology_key] = -1.0 * total_charge
 
                     # Apply increments to "orientation" atoms
```

Comment:
Does it make sense to log this? Since it will change the charges coming out. I can imagine being very confused that the charges are completely different to what the logs say they should be because they were modified by a charge increment from a virtual site.

Reply:
Probably - but this is already a mega-PR and should be dealt with at the intersection of this and #1070